README.rst

```diff
@@ -17,6 +17,17 @@ installed, you can run YOSO by simply executing ``python3 yoso.py``.
 Alternatively you can install YOSO with the `pip <https://pip.pypa.io>`_ package manager
 (which can be called ``pip3`` for Python 3).
 
+Installation
+============
+
+On Linux.
+
+1. apt-get install python-pip3
+2. pip3 install setuptools
+3. pip3 install pyqt5
+4. python3 setup.py install
+5. yoso
```

Comment:
This is too specific and out of scope. There are a lot of OSes out there, Linux-based, Solaris-based, Windows, etc. Users should know on their own how to install or distribute software for their specific environments.

I develop in Debian, and I find Debian's Qt5 more attractive than the one installed via pip. Because it uses GTK dialogs in my XFCE.

P. S. [Ubuntu is not Linux](https://www.reddit.com/r/Ubuntu/comments/1c4xx9/ubuntu_is_not_linux/) :)